packages/blade-mcp/knowledgebase/components/Card.md

```diff
@@ -325,3 +325,267 @@ const InteractiveCardExample = () => {
   );
 };
 ```
+
+### Metric Card
+A card displaying metrics with dynamic data visualization, hover effects, and responsive layout. Shows how to combine Card with data display components.
+
+```tsx
+import {
+  Card,
+  CardHeader,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+  CardHeaderLink,
+  CardHeaderBadge,
+  CardBody,
+  Box,
+  Text,
+  Amount,
+  ArrowSquareUpIcon,
+  ArrowRightIcon
+} from '@razorpay/blade/components';
+
+const MetricCard = () => {
+  const isMobile = window.innerWidth < 768;
+
+  return (
+    <Card
+      backgroundColor="surface.background.gray.intense"
+      maxWidth="500px"
+      minWidth="300px"
+      padding="spacing.5"
+      size="medium"
+    >
+      <CardHeader showDivider={false}>
+        <CardHeaderLeading
+          title={isMobile ? 'TPV' : 'Total Payment Volume'}
+          subtitle={isMobile ? 'TPV for the current month' : 'Total Payment Volume for the current month'}
+        />
+        <CardHeaderTrailing
+          visual={
+            isMobile ? (
+              <CardHeaderLink href="/" icon={ArrowRightIcon} iconPosition="right">
+                Chart settings
+              </CardHeaderLink>
+            ) : (
+              <CardHeaderBadge color="positive">New</CardHeaderBadge>
+            )
+          }
+        />
+      </CardHeader>
+      <CardBody>
+        <Box display="flex" flexDirection={isMobile ? 'row' : 'column'} gap="spacing.5">
+          <Box display="flex" flexDirection="column" justifyContent="flex-end">
+            <Box display="flex" flexDirection="row" gap="spacing.3" alignItems="center">
+              <Amount
+                value={1000}
+                color="surface.text.gray.normal"
+                weight="semibold"
+                size="2xlarge"
+                type="heading"
+              />
+              <Box display="flex" flexDirection="row" gap="spacing.1" alignItems="center">
+                <ArrowSquareUpIcon color="interactive.icon.positive.normal" />
+                <Text color="interactive.text.positive.normal">12</Text>
+              </Box>
+            </Box>
+          </Box>
+          {/* Add your chart/graph component here */}
+        </Box>
+      </CardBody>
+    </Card>
+  );
+};
+```
+
+### Multi-Selectable Product Cards
```

Comment:
Why is this example needed? we have interactive card example right?

Reply:
so their are 2 example , one is interactive. their we don't show checkbox, 
but the example i added here we are showing checkbox

Reply:
have changed it's tittle